manifests/a/Astronomer/Astro/1.34.1/Astronomer.Astro.installer.yaml

```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Astronomer.Astro
+PackageVersion: 1.34.1
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Redhat.Podman
+InstallerLocale: en-US
+InstallerType: portable
+UpgradeBehavior: install
+Commands:
+  - astro
+ReleaseDate: 2025-05-27
+Installers:
+  - Architecture: x64
+    InstallerUrl: https://github.com/astronomer/astro-cli/releases/download/v1.34.1/astro_1.34.1_windows_amd64.exe
+    InstallerSha256: A67A6783B3D699C6AD2BFF88462A567D47A13EF01AF7D83D4EAE1EE6B8C51925
+  - Architecture: x86
+    InstallerUrl: https://github.com/astronomer/astro-cli/releases/download/v1.34.1/astro_1.34.1_windows_386.exe
+    InstallerSha256: 8C0D2446943E0A9EED1733EA402B413D9EEADB9B461E388AE990395697C7FB05
```

Comment:
```suggestion
```
This needs to be removed because Redhat.Podman doesn't have x86 installers so it will fail on x86 operating systems.